Link to SAP Baseline Monitoring Alerts added to Operational Baseline section of SAP LZ (#1346)
MicrosoftDocs/cloud-adoption-framework: docs/scenarios/sap/eslz-management-and-monitoring.md

Here are some key design considerations and recommendations for monitoring and managing SAP on Azure:

- Use [Azure Monitor for SAP solutions](/azure/sap/monitor/about-azure-monitor-sap-solutions) to monitor your SAP workloads on Azure. Consider supplementing Azure Monitor for SAP solutions with SAP Solution Manager. You can add Application Insights to extract all necessary telemetry data while generating alerts and support tickets when telemetry indicates a potential error.

> - Configure baseline alerts for SAP [SAP on Azure Baseline Alerts](https://azure.github.io/azure-monitor-baseline-alerts/patterns/specialized/sap/)

Reviewer: ```suggestion
- Monitor the SAP environment using the [SAP on Azure Baseline Alerts](https://azure.github.io/azure-monitor-baseline-alerts/patterns/specialized/sap/)
```
